### ISSUE TITLE
PDF stamp and WaterMark - Not showing as expected for PDFs generated from chrome

### ISSUE THREAD

**Issue by ekanna** [hhrutter/pdfcpu#39]
Hi,

For pdf generated from excel, stamp and watermark is showing as expected. 
But pdf generated from chrome browser "print save as PDF" option, stamp and watermark  is not showing properly.

Please find attached PDF files.

[chrome.pdf](https://github.com/hhrutter/pdfcpu/files/2382576/g1.pdf)
[chrome_new.pdf](https://github.com/hhrutter/pdfcpu/files/2382577/g1_new.pdf)
[excel.pdf](https://github.com/hhrutter/pdfcpu/files/2382579/excel.pdf)
[excel_new.pdf](https://github.com/hhrutter/pdfcpu/files/2382580/excel_new.pdf)

Thanks

**Comment by hhrutter** [comment]
Which OS and Chrome version are you using?

I am using the latest Chrome (69.0.3497.92) on MacOS and it looks good to me.
(see the attached [mg_new.pdf](https://github.com/hhrutter/pdfcpu/files/2386134/mg_new.pdf))

**Comment by ekanna** [comment]
MacOS version 10.13.3
Chrome latest version 69.0.3497.92

I have tried again but getting same result. 
I have noticed another new issue. Please find attached pdfs.

[github.pdf](https://github.com/hhrutter/pdfcpu/files/2386271/github.pdf)
[github_new.pdf](https://github.com/hhrutter/pdfcpu/files/2386272/github_new.pdf)

Thanks

**Comment by hhrutter** [comment]
You are right. There is definitely something weird going on.

**Comment by hhrutter** [comment]
This is related to Type3 font usage.

 I am having a hard time reproducing your PDFs and need your help:

Inspecting your PDFs they show: 

```
Producer: Skia/PDF m68
Creator: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36
```

The PDFs  I create with Chrome on my Mac all show:

```
Producer: Mac OS X 10.13.6 Quartz PDFContext
Creator: Chrome
```

I am wondering what's different here.
How exactly do you generate your PDF out of Chrome?
There are several options like _Open PDF in Preview_, _Print using system dialogue_ and from there _Open in Preview_ and _Save as PDF_

Any PDF produced by Mac OS X 10.13.6 Quartz PDFContext does not cause any problem with pdfcpu for me.

**Comment by ekanna** [comment]
Screenshots  of how i generate PDF. And also find attached PDF generated

<img width="1440" alt="screen shot 2018-09-24 at 12 34 02 pm" src="https://user-images.githubusercontent.com/248011/45940546-61985280-bff7-11e8-89f7-4d965f66021a.png">
<img width="1440" alt="screen shot 2018-09-24 at 12 34 16 pm" src="https://user-images.githubusercontent.com/248011/45940567-71179b80-bff7-11e8-806d-84bef074cb43.png">

[PDF stamp and WaterMark - Not showing as expected for PDFs generated from chrome · Issue #39 · hhrutter_pdfcpu.pdf](https://github.com/hhrutter/pdfcpu/files/2409940/PDF.stamp.and.WaterMark.-.Not.showing.as.expected.for.PDFs.generated.from.chrome.Issue.39.hhrutter_pdfcpu.pdf)

Thanks
Koti

**Comment by hhrutter** [comment]
Sure. Open the PDF with Preview and then Show Inspector.

**Comment by hhrutter** [comment]
fixed with v0.1.18
